packages/arb-token-bridge-ui/src/hooks/CCTP/useUpdateUSDCBalances.ts

```diff
@@ -44,25 +27,61 @@ export function useUpdateUSDCBalances({
     walletAddress
   })
 
-  const updateUSDCBalances = useCallback(
-    (address: Address | string) => {
-      const l1Address = getL1AddressFromAddress(address)
+  const updateUSDCBalances = useCallback(async () => {
+    const { isEthereumMainnet, isSepolia, isArbitrumOne, isArbitrumSepolia } =
+      isNetwork(parentChain.id)
+
+    let parentChainUsdcAddress, childChainUsdcAddress: string | undefined
+
+    if (isEthereumMainnet || isSepolia) {
```

Comment:
NIT: `isEthereumMainnetOrTestnet` is clearer on the intentions IMO

Reply:
yes but we wouldn't want this case to be hit e.g. for Holesky